init/systemd/ipa.sysusers.redhat.conf

```diff
@@ -0,0 +1 @@
+ipa.sysusers.base.conf
```

Comment:
I created the symbolic links with the same login as platforms in ipaplatform use for inheritance but you're right there's no real platform redhat and we don't need sysusers.conf file for it.